wakuv2/nwaku.go

```diff
@@ -0,0 +1,58 @@
+package wakuv2
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+)
+
+type NwakuInfo struct {
+	ListenAddresses []string `json:"listenAddresses"`
+	EnrUri          string   `json:"enrUri"`
+}
+
+func GetNwakuInfo(host *string, port *int) (NwakuInfo, error) {
```

Comment:
This can be probably used for the tests that are currently skipped due to requiring storeV3

Reply:
Yeah! Also, something we could do now is to disable the mailserver_cycle so we dont use the fleet in some of the tests.